www/docs/usage/commandline.md

```diff
@@ -53,45 +56,78 @@ The most recently selected versions are presented at the top of the dropdown.
 
 1. Show the latest implicit stable version.
-2. Ex: `tfswitch -S 0.13` or `tfswitch --show-latest-stable 0.13` shows 0.13.6 (latest) version.
+2. Ex: `tfswitch -S 0.13` or `tfswitch --show-latest-stable 0.13` shows 0.13.6
+   (latest) version.
 3. Hit **Enter** to show.
 
 ## Show latest implicit version for beta, alpha and release candidates(rc)
 
 1. Show the latest implicit prerelease version.
-2. Ex: `tfswitch -P 0.13` or `tfswitch --show-latest-pre 0.13` shows 0.13.0-rc1 (latest) version.
+2. Ex: `tfswitch -P 0.13` or `tfswitch --show-latest-pre 0.13` shows 0.13.0-rc1
+   (latest) version.
 3. Hit **Enter** to show.
 
 ## Use custom mirror
 
-To install from a remote mirror other than the default (<https://releases.hashicorp.com/terraform>). Use the `-m` or `--mirror` parameter.
+To install from a remote mirror other than the default
+(<https://releases.hashicorp.com/terraform>). Use the `-m` or `--mirror`
+parameter.
 
 ```bash
 tfswitch --mirror https://example.jfrog.io/artifactory/hashicorp`
 ```
 
 ## Install to non-default location
 
-By default `tfswitch` will download the Terraform binary to the user home directory under this path: `$HOME/.terraform.versions`
+By default `tfswitch` will download the Terraform binary to the user home
+directory under this path: `$HOME/.terraform.versions`
 
-If you want to install the binaries outside of the home directory then you can provide the `-i` or `--install` to install Terraform binaries to a non-standard path. Useful if you want to install versions of Terraform that can be shared with multiple users.
+If you want to install the binaries outside of the home directory then you can
+provide the `-i` or `--install` to install Terraform binaries to a non-standard
+path. Useful if you want to install versions of Terraform that can be shared
+with multiple users.
 
-The Terraform binaries will then be placed in the directory `.terraform.versions` under the custom install path e.g. `/opt/terraform/.terraform.versions`
+The Terraform binaries will then be placed in the directory
+`.terraform.versions` under the custom install path e.g.
+`/opt/terraform/.terraform.versions`
 
 ```bash
 tfswitch -i /opt/terraform
 ```
 
-**NOTE**: The directory passed in `-i`/`--install` must be created before running `tfswitch`
+**NOTE**: The directory passed in `-i`/`--install` must be created before
+running `tfswitch`
 
 ## Install binary for non-default architecture
 
-By default `tfswitch` will download the binary for the architecture of the host machine.
+By default `tfswitch` will download the binary for the architecture of the host
+machine.
 
-If you want to download the binary for non-default CPU architecture then you can provide the `-A` or `--arch` command line argument to download binaries for custom CPU architecture. Useful if you need to override binary architecture for whatever reason.
+If you want to download the binary for non-default CPU architecture then you
```

Comment:
Not sure what's being changed in these lines - but seeing this, perhaps `non-default` architecture doesn't sound quite right - maybe `a CPU architecture that doens't match the host`?